<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderQuery</class>
 <widget class="QDialog" name="OrderQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>509</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>581</width>
    <height>509</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>581</width>
    <height>509</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Logo{
	image: url(:/images/Logo_Order.png);
}
#Point{
	
	image: url(:/images/Point.png);
}
#CardBackground{
	
	border-image: url(:/images/CardBackground.png);
	
}
#CardBackground:hover{
	border-image: url(:/images/CardBackground2.png);

}
#Background{
	background-color:rgb(247,247,247);
}
#Sentence{
	background-color:rgb(247,247,247);
	border:none;
}
#BackgroundLogo{
	
	image: url(:/images/BackgroundLogo_Order.png);
}
#Sentence{
	
	image: url(:/images/Sentence.jpg);
}
#TurnLeft{
	border-radius:4px;
}
#Close_Btn{
	border-radius:7px;
	border:none;
	background-color:none;
}
#Close_Btn:hover{
	border:1px solids rgb(175,175,175);
	background-color:rgb(255,0,0);
	border-radius:7px;
}
#NextCard{
	border-image: url(:/images/NextCard.png);
}</string>
  </property>
  <widget class="QLabel" name="MyOrder">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>STKaiti</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>我的订单</string>
   </property>
  </widget>
  <widget class="QLabel" name="CityStart">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>3</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>STKaiti</family>
     <pointsize>24</pointsize>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>西安</string>
   </property>
  </widget>
  <widget class="QLabel" name="CityEnd">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>STKaiti</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>上海</string>
   </property>
  </widget>
  <widget class="QLabel" name="Point">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Logo">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="TimeStart">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>STKaiti</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string> 10:26</string>
   </property>
  </widget>
  <widget class="QLabel" name="TimeEnd">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>STKaiti</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>13:42</string>
   </property>
  </widget>
  <widget class="QLabel" name="FromTo">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>STKaiti</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>至</string>
   </property>
  </widget>
  <widget class="QLabel" name="PlaneNumber">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>STKaiti</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MU2345</string>
   </property>
  </widget>
  <widget class="QLabel" name="Money_Label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>110</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>STKaiti</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(250,126,8);</string>
   </property>
   <property name="text">
    <string>￥</string>
   </property>
  </widget>
  <widget class="QLabel" name="Money">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>STKaiti</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(250,126,8);</string>
   </property>
   <property name="text">
    <string>367</string>
   </property>
  </widget>
  <widget class="QLabel" name="BackgroundLogo">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>361</width>
     <height>421</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Sentence">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>261</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Background">
   <property name="geometry">
    <rect>
     <x>-60</x>
     <y>-70</y>
     <width>711</width>
     <height>831</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>711</width>
     <height>831</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>711</width>
     <height>831</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Close_Btn">
   <property name="geometry">
    <rect>
     <x>498</x>
     <y>90</y>
     <width>15</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="Data">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>STKaiti</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2018/12/12</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CardBackground">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>461</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>Background</zorder>
  <zorder>BackgroundLogo</zorder>
  <zorder>CardBackground</zorder>
  <zorder>Sentence</zorder>
  <zorder>MyOrder</zorder>
  <zorder>CityStart</zorder>
  <zorder>CityEnd</zorder>
  <zorder>Point</zorder>
  <zorder>Logo</zorder>
  <zorder>TimeStart</zorder>
  <zorder>TimeEnd</zorder>
  <zorder>FromTo</zorder>
  <zorder>PlaneNumber</zorder>
  <zorder>Money_Label</zorder>
  <zorder>Money</zorder>
  <zorder>Close_Btn</zorder>
  <zorder>Data</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
